course_discovery/apps/course_metadata/management/commands/update_course_run_in_salesforce.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from django.core.management import BaseCommand, CommandError
+from django.utils.translation import ugettext as _
+
+from course_discovery.apps.course_metadata.choices import CourseRunStatus
+from course_discovery.apps.course_metadata.models import CourseRun
+from course_discovery.apps.course_metadata.salesforce import SalesforceUtil
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Based on course run go_live dates, publish any runs that are due'
+
+    def handle(self, *args, **options):
+        failed_course_runs = []
+        course_runs = CourseRun.objects.filter(status=CourseRunStatus.Published)
```

Comment:
I guess instead of published-only course runs, we filter only on non-draft versions.
```suggestion
        course_runs = CourseRun.objects.filter(draft=False)
```